### DISCUSSION TITLE
Where's the admin password documented?

### DISCUSSION THREAD

**Discussion by Tony20221** [serenity-is/Serenity#6604] [Help]
I installed the app and it's showing the login screen. I know the user name is admin because I looked at the users table.
I know the password is serene from a post in an issue.
Where are the user name and password documented??

**Comment by VictorTomaili** [comment]
Use the demo password
https://serenity.is/demo

**Reply by Tony20221** [reply]
What is the password!?? The password is masked and I can't find it in the docs. How am I supposed to know it's on some demo site??

**Reply by minhhungit** [reply]
@Tony20221 
![image](https://user-images.githubusercontent.com/2279508/205458336-8584a1f2-97e0-404e-9065-1c0c6cbc71c6.png)

**Reply by Tony20221** [reply]
The password needs to be in the documentation or the documentation needs to tell users how to find it. How am I supposed to know that the password is on some demo site when I am able to run the app locally!

**Reply by VictorTomaili** [reply]
By reading the source code in your local environment, maybe?

![image](https://user-images.githubusercontent.com/2023440/205483322-5ad6bf54-6c29-4aa2-ae1c-3d76b7be0414.png)

**Comment by volkanceylan** [comment]
Before complaining about this:

> The password needs to be in the documentation or the documentation needs to tell users how to find it.

Try actually reading the documentation:

https://serenity.is/docs/getting_started/starting_serene

<img width="693" alt="image" src="https://user-images.githubusercontent.com/752470/205495477-8cd367be-56c7-4683-a195-3bd43b127ebd.png">
